### PR TITLE
[keras/optimizers/legacy] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

File: keras/optimizers/legacy/optimizer_v2.py [hunk 1]
```diff
@@ -692,8 +692,8 @@ def apply_gradients(
 
         Args:
           grads_and_vars: List of (gradient, variable) pairs.
-          name: Optional name for the returned operation. Default to the name
-            passed to the `Optimizer` constructor.
+          name: Optional name for the returned operation. When None, uses the
```
Comment (qlzh727) [inline keras/optimizers/legacy/optimizer_v2.py:695]: Please quote the None her.

File: keras/optimizers/legacy_learning_rate_decay.py [hunk 2]
```diff
@@ -264,9 +264,10 @@ def polynomial_decay(
       end_learning_rate: A scalar `float32` or `float64` `Tensor` or a Python
         number.  The minimal end learning rate.
       power: A scalar `float32` or `float64` `Tensor` or a Python number.  The
-        power of the polynomial. Defaults to linear, 1.0.
-      cycle: A boolean, whether or not it should cycle beyond decay_steps.
-      name: String.  Optional name of the operation. Defaults to
+        power of the polynomial. Linear is default. Defaults to `1.0`.
```
Comment (qlzh727) [inline keras/optimizers/legacy_learning_rate_decay.py:267]: Let's remove `Linear is default.`

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/optimizers/legacy_learning_rate_decay.py (hunk 2 above)
Comment (SamuelMarks) [reply keras/optimizers/legacy_learning_rate_decay.py:267]: Fixed

File: keras/optimizers/legacy/optimizer_v2.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/optimizers/legacy/optimizer_v2.py:695]: Fixed

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**